packages/fxa-settings/src/components/ConnectedServices/index.tsx

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from 'react';
+import { AlertBar } from '../AlertBar';
+import { ButtonIconReload } from '../ButtonIcon';
+import { useAlertBar } from 'fxa-settings/src/lib/hooks';
+import { useAccount, useLazyAccount } from '../../models';
+import { LinkExternal } from 'fxa-react/components/LinkExternal';
+
+const UTM_PARAMS =
+  '?utm_source=accounts.firefox.com&utm_medium=referral&utm_campaign=fxa-devices';
+const DEVICES_SUPPORT_URL =
+  'https://support.mozilla.org/kb/fxa-managing-devices' + UTM_PARAMS;
+
+export const Service = () => {
+  return (
+    <section
+      className="my-1"
+      id="service"
+      data-testid="settings-connected-service"
+    >
+      <div className="border-2 border-solid border-grey-100 rounded flex justify-around items-center">
+        <span>icon</span>
+
+        <div className="flex flex-col">
+          <h2>Service Name</h2>
```

Comment:
I think this is actually `12px` according to the comps, as well as the location.

Reply:
I'm going to ignore these comments for the unit row for individual services since it's going to be addressed in https://jira.mozilla.com/browse/FXA-1757